infra/modules/azuread-local-cert/local-cert-standalone.sh

```diff
@@ -6,12 +6,14 @@ SUBJECT=$1
 TTL=$2
 AZURE_TOOL=$3
```

Comment:
could use `tr` to do the uppercase here in case it is really needed (not sure what the call comes from)

https://scriptingosx.com/2019/12/upper-or-lower-casing-strings-in-bash-and-zsh/

Reply:
people have to do it themselves; it's only legacy customers using ad certificates at this point, and of those I think perhaps 2 do this outside of terraform.